crates/figma_import/src/toolkit_schema.rs

```diff
@@ -122,6 +124,9 @@ pub enum RenderMethod {
 /// Represents a toolkit View (like a Composable).
 #[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
 pub struct View {
+    // unique numeric id (not replicated within one DesignCompose file), which is used by the
+    // renderer when building a layout tree, or when mapping from an integer to a View.
+    pub unique_id: u16,
```

Comment:
I think this warrants a serialized document header version bump.

Reply:
Oops, that's right, I'll update the header version.

Reply:
Introducing DCF v17...